RUNBOOK.md

```diff
@@ -0,0 +1,153 @@
+
+## System characteristics
+
+### Hours of operation
+
+Available 24/7.
+
+### Infrastructure design
+
+The application runs on the [HMPPS Cloud Platform](https://user-guide.cloud-platform.service.justice.gov.uk/) within the `manage-recalls-<tier>` namespaces (where `tier` is `dev`, `preprod` or `prod`).
+
+**NOTE:** The service runs on the `live` cluster, not the deprecated `live-1` instance.
+
+The main application runs as a deployment named `manage-recalls-ui`. The application is made avialable externally from the cluster via an Ingress.
+
+See the `values-<tier>.yaml` files in the [helm_deploy](helm_deploy) directory for the current configuration of each tier.
+
+### Throttling and partial shutdown
+
+If there is an issue with the service where it is causing load on downstream services and it needs to be shutdown quickly the following command will reduce the number of pod replicas to zero:
+
+```
+kubectl -n manage-recalls-<tier> scale deployment manage-recalls-ui --replicas=0
+```
+
+We do not currently have a strategy in place to throttle requests.
+
+### Environmental differences
+
+Infrastructure wise, all three tiers are identical, but `prod` has the following differences:
+
+- It will have more pod replicas of the main application deployment.
+- As this is live data, you **must** be SC cleared to work on it or access the data held within it.
+
+### Tools
+
+See the [scripts](scripts) directory and the documentation held in [docs](docs) for more details.
+
+## Security and access control
+
+In order to gain access to the `manage-recalls-<tier>` namespaces in kubernetes you will need to be a member of the [ministryofjustice](https://github.com/orgs/ministryofjustice) github organisation and a member of the [ppud-replacement-devs](https://github.com/orgs/ministryofjustice/teams/ppud-replacement-devs) (github) team. Once joined, you should have access to the cluster within 24 hours.
+
+You will need to follow the [Cloud Platform User Guide](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/kubectl-config.html#how-to-use-kubectl-to-connect-to-the-cluster) to setup your access from there - use instructions for connecting to the `live` cluster.
+
+## System configuration
+
+### Configuration management
+
+- Infrastructure is configured via [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments).
+- Application configuration is via [helm_deploy](helm_deploy).
+
+### Secrets management
+
+Secrets are stored within the `manage-recalls-<tier>` namespaces in kubernetes.
+
+Secrets with information from [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments) will be managed via the terraform code in there.
+
+The contents of the `manage-recalls-ui` secret are handled in the following way:
+
+- `API_CLIENT_ID` - managed externally via tooling in the `hmpps-auth` project.
```

Comment:
This duplicates existing info in the readme's, could link from there to here, or vice versa? Otherwise we're maintaining two lists.

Reply:
Ah, not seen that in the README - will check and remove duplication.